src/components/DataTable/DataTable.less

```diff
@@ -0,0 +1,14 @@
+.lucid-DataTable {
+	.lucid-Table-density-extended {
+		.lucid-Table-Tbody {
+			.lucid-Table-Tr {
+				.lucid-Table-Td.lucid-DataTable-empty-cell {
```

Comment:
Yeah, agreed. I was having some issues with specificity initially on this so I might've gone a _little_ overboard.